src/components/team-table/time-cell/time-cell.component.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { TimeCellWrapper, TimeCellFragment, WorkTime } from './time-cell.styles';
+import TableCell from '../table-cell/table-cell.component';
+import CurrentTime from '../current-time/current-time.component';
+import { useDispatch, useSelector } from 'react-redux';
+import { pickUser } from '../../../redux/worktime-display/worktime-display.actions';
+
+const checkTimeIsWorkTime = (time, workTime) => {
```

Comment:
As far as I understand, the time unit of work time is hours, and hence users can't set minutes as a border of work time periods. Did I understand this correctly?